packages/store-engine-fs/src/index.ts

```diff
@@ -106,11 +106,7 @@ export class FileEngine implements CRUDEngine {
       }
 
       const filePath = this.resolvePath(tableName, primaryKey);
-      let newEntity: EntityType | string = entity;
-
-      if (typeof entity === 'object') {
-        newEntity = JSON.stringify(entity);
-      }
+      const newEntity: string = JSON.stringify(entity);
```

Comment:
```suggestion
let newEntity: string;
     if (typeof entity === 'object') {
        newEntity = JSON.stringify(entity);
      }else{
      newEntity = entity;

```
perhaps this works?

Reply:
I'd shorten it to
```
const newEntity: string = typeof entity === 'object' ? JSON.stringify(entity) : entity;
```
but also, maybe `... : entity.toString()`?